dart/lib/src/enricher/web_enricher_event_processor.dart

```diff
@@ -1,17 +1,11 @@
 import 'dart:async';
-import 'dart:html' as html show window, Window;
+import 'dart:html' as html;
 
 import '../event_processor.dart';
 import '../protocol.dart';
 import '../sentry_options.dart';
 
-EventProcessor enricherEventProcessor(SentryOptions options) {
-  return WebEnricherEventProcessor(
-    html.window,
-    options,
-  );
-}
```

Comment:
@ueman do you remember why this isn't used?

Reply:
It is used, but imported via conditional import, maybe that's the problem?

https://github.com/getsentry/sentry-dart/blob/eecbbcabbbe00a5dcc7d42cd6334f771a20b547f/dart/lib/src/enricher/enricher_event_processor.dart#L1-L8